### PR TITLE
Update release workflow

### PR DESCRIPTION
**Pull request by granluo**
This update is prepared for released candidate nightly testing.

### CODE REVIEW

File: scripts/release_testing_setup.sh [hunk 1]
```diff
@@ -14,28 +14,41 @@
...
-# so error is allowed here.
-git push origin --delete "${tag_version}"
-set -e
-git tag -f -a "${tag_version}" -m "release testing"
-git push origin "${tag_version}"
-
-# Update source and tag, e.g.  ":tag => 'CocoaPods-' + s.version.to_s" to
-# ":tag => test"
-sed  -i "" "s/\s*:tag.*/:tag => '${tag_version}'/" *.podspec
+if [ -n "$tag_version" ]; then
```
Comment (granluo) [inline scripts/release_testing_setup.sh:39]: will only push a tag if tag_version is not null.

File: scripts/setup_quickstart.sh [hunk 2]
```diff
@@ -40,11 +40,17 @@ if check_secrets || [[ ${SAMPLE} == "installations" ]]; then
 
   git clone https://github.com/firebase/quickstart-ios.git
   $scripts_dir/localize_podfile.swift quickstart-ios/"$SAMPLE"/Podfile "$RELEASE_TESTING"
-  if [ ! -z "$RELEASE_TESTING" ]; then
+  if [ "$RELEASE_TESTING" == "nightly_release_testing" ]; then
```
Comment (granluo) [inline scripts/setup_quickstart.sh:43]: update the flag of nightly release testing to "nightly_release_testing" and the flag for release candidate to  "prerelease_testing". Prerelease testing will be set up in another pr.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Update SpecsTesting repo setup
       run: |
          ossbotaccess=`cat oss-bot-access.txt`
-         BOT_TOKEN="${ossbotaccess}" nightly_test_version="${nightly_version}" scripts/release_testing_setup.sh
+         BOT_TOKEN="${ossbotaccess}" test_version="${nightly_version}" scripts/release_testing_setup.sh nightly_testing
```
Comment (paulb777) [inline .github/workflows/release.yml:34]: Where is test_version used?

**Review by paulb777 — APPROVED** [review]

File: .github/workflows/release.yml (hunk 3 above)
Comment (granluo) [reply .github/workflows/release.yml:34]: It is used in release_testing_setup.sh to update a tag.
https://github.com/firebase/firebase-ios-sdk/blob/38af2d564336b06efdd24920de08230364d4957b/scripts/release_testing_setup.sh#L31-L35

Comment (granluo) [reply .github/workflows/release.yml:34]: test_version should be updated in the workflow and testing tags need to be removed manually from repo at this moment.
